### ISSUE TITLE
Emit event when processing is complete

### ISSUE THREAD

**Issue by JavaScriptDude** [danielkrainas/node-netstat#1]
In order to use your utility programatically, the consumer needs to know when the parsing is complete. I recommend adding to netstat.js::stopParsing() a call to a callback defined by the consumer in options.

For instance calling options.parsingDone().

Thanks for the cool utility.

**Comment by danielkrainas** [comment]
Thanks for the feedback! The change has been published in the v0.9.0 release along with documentation. 

Usage: 

``` js
netstat({
    done: function (err) {
        // command completed
    }
}, /* line handler */);
```

**Comment by JavaScriptDude** [comment]
Thanks. I've got one more. I'll post...
